packages/accordions/src/elements/accordion/components/Label.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { forwardRef, HTMLAttributes } from 'react';
+import { StyledButton } from '../../../styled';
+import { useAccordionContext, useHeaderContext } from '../../../utils';
+
+export const Label = forwardRef<HTMLButtonElement, HTMLAttributes<HTMLButtonElement>>(
```

Comment:
Would it be more clear if this was renamed from "Label" to "Button" since it is the header button? I kept "Label" since that was a suggestion during a pair session. Open to changing this.

Reply:
Would it be more clear if this component was renamed from "Label" to "Button" since it is the header button? I kept "Label" since that was a suggestion during a pair session. Open to changing this.